unix/requirements.txt

```diff
@@ -4,4 +4,4 @@
 #
 PySDL2
 Pillow
-
```

Comment:
maybe you had an install failure? `numpy` is required by `zbar-py` which is listed in `testing/requirements.txt`

Reply:
probably a mistake from the 
cd unix

Testing now.